src/constants/networks/matic.ts

```diff
@@ -63,9 +63,10 @@ const maticInfo: EVMNetworkInfo = {
   },
   elastic: {
     // client: createClient('https://api.thegraph.com/subgraphs/name/kybernetwork/kyberswap-elastic-matic'),
-    client: createClient(
-      'https://polygon-graph.kyberengineering.io/subgraphs/name/kybernetwork/kyberswap-elastic-polygon',
-    ),
+    // client: createClient(
+    //   'https://polygon-graph.kyberengineering.io/subgraphs/name/kybernetwork/kyberswap-elastic-polygon',
+    // ),
+    client: createClient('https://api.thegraph.com/subgraphs/name/viet-nv/kyberswap-elastic-matic'),
```

Comment:
clgt

Reply:
Just deploy all new subgraph to my account for testing, when QA verifies that it's ok, i will deploy on kyber account using graft mode, this will safe and save time for us

Reply:
:pepe-ok-hand: